src/content/docs/usage/Getting_started/introduction.md

```diff
@@ -7,90 +7,92 @@ parentWeight: 10
 - Portability to run your pipelines anywhere: laptop, cluster, or cloud.
-- Reproducibility of analyses independent of time and computing platform.
+- Reproducibility of analyses, independent of time and computing platform.
 
 Whether your pipeline is a simple BLAST execution or a complex genome annotation pipeline, you can build it with Nextflow.
 
 ## How to run a pipeline
 
-Nextflow works best when you have an active internet connection, as it is able to fetch all pipeline requirements. However, if you need to run offline, see [_Running offline_](offline.md).
+Nextflow works best with an active internet connection, as it is able to fetch all pipeline requirements. See [Running offline](offline.md) if you need to run Nextflow pipelines without an internet connection.
 
-1. First, make sure that you have all software dependencies installed (Nextflow + Docker / Singularity / Conda). See the [installation docs](installation.md) for more information.
+To run a pipeline:
 
-   - Try running the Nextflow "hello world" example to make sure that the tools are working properly:
+1. All software dependencies must be installed (Nextflow + Docker / Singularity / Conda). See [Dependency installation](installation.md) for more information.
+
+   - Run Nextflow's "hello world" example to confirm all tools are installed and operational:
 
      ```bash
      nextflow run hello
      ```
 
-2. Choose a pipeline to run. See the available pipelines at [https://nf-co.re/pipelines](https://nf-co.re/pipelines). If you have [https://nf-core/tools](https://nf-co.re/tools) installed, run `nf-core list`.
+2. Choose a pipeline to run. See the available [nf-core pipelines](https://nf-co.re/pipelines). If you have [nf-core tools](https://nf-co.re/tools) installed, run `nf-core list`.
 
-3. Configure Nextflow to run on your system.
+3. Configure Nextflow to run on your system:
 
-   - The simplest way to run is with `-profile docker` (or `singularity`) which instructs Nextflow to execute jobs locally using Docker to fulfill the software dependencies.
+   - The simplest way to run is with `-profile docker` (or `singularity`). This instructs Nextflow to execute jobs locally, with Docker (or Singularity) to fulfill software dependencies.
 
    - Conda is also supported with `-profile conda`. However, this option is not recommended, as reproducibility of results can't be guaranteed without containerization.
 
-   - If you are a member of one of the institutions listed in the [documentation](https://github.com/nf-core/configs#documentation), please use the custom config file [nf-core/configs/conf](https://github.com/nf-core/configs/tree/master/conf) that has been created for your institution.
+   - If you are a member of one of the [listed institutions](https://github.com/nf-core/configs#documentation), use the [custom config file](https://github.com/nf-core/configs/tree/master/conf) created for your institution.
```

Comment:
```suggestion
   - If you are a member of one of the [listed institutions](https://github.com/nf-core/configs#documentation), use the [institutional config file](https://github.com/nf-core/configs/tree/master/conf) created for your institution.
```

I know this puts 'institution' too many times, but we actually use 'custom' config in most other places to refer to single user configs for specific runs/projects

Alternative word could be 'infrastructure' config